src/range_proof/mod.rs

```diff
@@ -369,13 +371,12 @@ fn delta(n: usize, m: usize, y: &Scalar, z: &Scalar) -> Scalar {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use rand::rngs::OsRng;
 
-    use generators::PedersenGenerators;
+    use generators::PedersenGens;
 
     #[test]
     fn test_delta() {
-        let mut rng = OsRng::new().unwrap();
+        let mut rng = rand::thread_rng();
```

Comment:
what's the motivation for changing the source of randomness?

Reply:
There's no really good reason -- I removed the `OsRng` that was used in other parts of the test code while changing that code, then removed the `use` declaration that brought it into scope, then realized that the `test_delta` function was using it, and didn't want to re-add it, so I just made everything use `thread_rng`